exercise3/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+vcs import < src/ros2.repos src
+sudo apt-get update
+rosdep update
+rosdep install --from-paths src --ignore-src -y
```

Comment:
This doesn't actually work for the docker and raspberry pi setup as we assume to be working offline. Can you list the dependencies that we need for this and I can add it into the docker file?

Reply:
List generated for exercises 3 and 4. I assume most are already included in the image

```
oren@oren-mc1040:~/git/roscon-2023-realtime-workshop/exercise3$ rosdep install --simulate --reinstall --from-path src
#[apt] Installation commands:
  sudo -H apt-get install ros-humble-rmw
  sudo -H apt-get install ros-humble-lifecycle-msgs
  sudo -H apt-get install ros-humble-rclcpp
  sudo -H apt-get install ros-humble-rcl-lifecycle
  sudo -H apt-get install ros-humble-rosidl-typesupport-cpp
  sudo -H apt-get install ros-humble-ament-cmake-gtest
  sudo -H apt-get install ros-humble-ament-lint-auto
  sudo -H apt-get install ros-humble-ament-lint-common
  sudo -H apt-get install ros-humble-mimick-vendor
  sudo -H apt-get install ros-humble-performance-test-fixture
  sudo -H apt-get install ros-humble-rcpputils
  sudo -H apt-get install ros-humble-rcutils
  sudo -H apt-get install ros-humble-test-msgs
  sudo -H apt-get install ros-humble-ament-cmake-ros
  sudo -H apt-get install ros-humble-ament-index-cpp
  sudo -H apt-get install ros-humble-builtin-interfaces
  sudo -H apt-get install ros-humble-rcl-interfaces
  sudo -H apt-get install ros-humble-rosgraph-msgs
  sudo -H apt-get install ros-humble-rosidl-runtime-cpp
  sudo -H apt-get install ros-humble-rosidl-typesupport-c
  sudo -H apt-get install ros-humble-libstatistics-collector
  sudo -H apt-get install ros-humble-rcl
  sudo -H apt-get install ros-humble-rcl-yaml-param-parser
  sudo -H apt-get install ros-humble-statistics-msgs
  sudo -H apt-get install ros-humble-tracetools
  sudo -H apt-get install ros-humble-ament-cmake-gmock
  sudo -H apt-get install ros-humble-ament-cmake-google-benchmark
  sudo -H apt-get install ros-humble-rmw-implementation-cmake
  sudo -H apt-get install ros-humble-rosidl-default-generators
  sudo -H apt-get install ros-humble-ament-cmake-gen-version-h
  sudo -H apt-get install python3-dev
  sudo -H apt-get install ros-humble-std-msgs
  sudo -H apt-get install ros-humble-ament-cmake
  sudo -H apt-get install ros-humble-class-loader
  sudo -H apt-get install ros-humble-composition-interfaces
  sudo -H apt-get install ros-humble-launch-testing
  sudo -H apt-get install ros-humble-rosidl-runtime-c
  sudo -H apt-get install ros-humble-action-msgs
  sudo -H apt-get install ros-humble-rcl-action

oren@oren-mc1040:~/git/roscon-2023-realtime-workshop/exercise4$ rosdep install --simulate --reinstall --from-path src
#[apt] Installation commands:
  sudo -H apt-get install ros-humble-ament-lint-auto
  sudo -H apt-get install ros-humble-ament-lint-common
  sudo -H apt-get install ros-humble-ament-cmake
```